tests/core/utilities/test_encoding.py

```diff
@@ -28,7 +23,10 @@
     JSONBaseProvider,
 )
 from web3.tools.hypothesis import (
+    example,
+    given,
     hexstr_strategy,
+    strategies as st,
```

Comment:
`given`, `example` and `strategies` should be imported directly from the `hypothesis` library.

Reply:
I think only 4 packages should from hypothesis are used 
So in this case only `hexer_strategy` should be taken from web3.tools.hypothesis